### PR TITLE
[zh-cn] Add blog 2024-08-12-feature-gates_client-go.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh-cn] Add blog 2024-08-12-feature-gates_client-go.md

### Issue

Closes: #

### CODE REVIEW

File: content/zh-cn/blog/_posts/2024-08-12-feature-gates_client-go.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+author: >
+ Ben Luddy (Red Hat),
+ Lukasz Szaszkiewicz (Red Hat)
+-->
+
+<!--
+Kubernetes components use on-off switches called _feature gates_ to manage the risk of adding a new feature.
+The feature gate mechanism is what enables incremental graduation of a feature through the stages Alpha, Beta, and GA.
+-->
+Kubernetes 组件使用称为“特性门控”的开关来管理添加新特性的风险。
```
Comment (leslie-tsang) [inline content/zh-cn/blog/_posts/2024-08-12-feature-gates_client-go.md:26]: It's strange to call `feature gates` as `特性门控`.
Depending on the context, how about calling it a `特性开关` instead. :)

File: content/zh-cn/blog/_posts/2024-08-12-feature-gates_client-go.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+<!--
+To replace the default feature gate provider, implement the Gates interface and call ReplaceFeatureGates 
+at package initialization time, as in this simple example:
+-->
+要替换默认的特性门控提供程序，请实现 Gates 接口并在包初始化时调用 ReplaceFeatureGates，如以下简单示例所示：
+
+```go
+import (
+ “k8s.io/client-go/features”
```
Comment (leslie-tsang) [inline content/zh-cn/blog/_posts/2024-08-12-feature-gates_client-go.md:178]:
```suggestion
    "k8s.io/client-go/features"
```
example code typo. :)

File: content/zh-cn/blog/_posts/2024-08-12-feature-gates_client-go.md (hunk 1 above)
Comment (my-git9) [reply content/zh-cn/blog/_posts/2024-08-12-feature-gates_client-go.md]: The translation into `特性门控` is a tacit understanding left by the previous document translation

File: content/zh-cn/blog/_posts/2024-08-12-feature-gates_client-go.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+With the introduction of feature gates in client-go v1.30, rolling out a new client-go feature has become safer and easier. 
+Users and developers can control the pace of their own adoption of client-go features. 
+The work of Kubernetes contributors is streamlined by having a common mechanism for graduating features that span both sides of the Kubernetes API boundary.
+-->
+## 总结
+
+随着 client-go v1.30 中特性门控的引入，推出新的 client-go 特性变得更加安全、简单。
+用户和开发人员可以控制自己采用 client-go 特性的步伐。
+通过采用一种通用机制来分级跨越 Kubernetes API 边界两侧的特性，Kubernetes 贡献者的工作得到了简化。
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2024-08-12-feature-gates_client-go.md:212]:
```suggestion
通过为跨 Kubernetes API 边界两侧的特性提供一种通用的培育机制，Kubernetes 贡献者的工作得到了简化。
```

**Pull request merged**